ratpack-codahale-metrics/src/main/java/ratpack/codahale/metrics/CodaHaleMetricsModule.java

```diff
@@ -651,6 +680,81 @@ public Csv reportDirectory(File reportDirectory) {
       }
     }
 
+    public static class Graphite {
```

Comment:
I'm wondering whether we should bring these out into their own class files, this module class is getting pretty big.  What do you think?